run_hyper.py

```diff
@@ -26,12 +26,14 @@ def main():
     args, _ = parser.parse_known_args()
 
     # plz set algo='exhaustive' to use exhaustive search, in this case, max_evals is auto set
+    # in other case, max_evals needs to be set manually
     config_file_list = (
         args.config_files.strip().split(" ") if args.config_files else None
     )
     hp = HyperTuning(
         objective_function,
         algo="exhaustive",
+        early_stop=10,
```

Comment:
@Wicknight 
`early_stop` is default parameter like `max_evals`, maybe we can remove it when defining function or add `max_evals` to stay the same.

p.s., your local changes to the following files would be overwritten by checkout:
	`recbole/trainer/hyper_tuning.py`
Please commit your changes or stash them so that code formatting can work.